### DISCUSSION TITLE
Add homebrew cask

### DISCUSSION THREAD

**Discussion by kzuraw** [LocalCan/LocalCanApp#1] [Feature requests]
It will be awesome to instal LocalCan via homebrew:
```shell
brew install --cask localcan
```

Related docs: https://docs.brew.sh/Adding-Software-to-Homebrew#casks

**Comment by Jaarson** [comment]
I really like that!

**Comment by paour** [comment]
I've submitted a PR. https://github.com/Homebrew/homebrew-cask/pull/182809

**Reply by paour** [reply]
The PR has been merged. `brew install localcan`.

**Reply by Jaarson** [reply]
Thank you so much @paour!
I posted an [update on X](https://x.com/LocalCanApp/status/1831265877615943898) and will add it to the website soon.
